src/components/Amalgamation/ResultingBusinessName.vue

```diff
@@ -85,6 +86,7 @@ import NameTranslations from '@/components/common/NameTranslations.vue'
   }
 })
 export default class ResultingBusinessName extends Mixins(AmalgamationMixin, NameRequestMixin) {
+  // @Getter(useStore) getAmalgamatingBusinesses!: Array<AmalgamatingBusinessIF>
```

Comment:
As in other files, this getter is used in this component but is actually imported via a mixin, but I like to see the getter declaration here anyway.